README.md

```diff
@@ -31,45 +92,71 @@ Finally install all of the packages.
 poetry install
 ```
 
+
+
+
+
+
+
 Project Organization
 ------------
 
+ 
+
     ├── data
-    │   ├── external         <- Data from third party sources.
-    │   ├── interim          <- Intermediate data that has been transformed.
-    │   ├── processed        <- The final, canonical data sets for modeling.
-    │   └── raw              <- The original, immutable data dump.
+    │   ├── external                        <- Data from third party sources.
+    │   ├── interim                         <- Intermediate data that has been transformed.
+    │   ├── processed                       <- The final, canonical data sets for modeling.
+    │   └── raw                             <- The original, immutable data dump.
     │
-    ├── discussion_agents    <- Source code for this project.
-    │   │
-    │   ├── memory           <- Memory-related modules.
-    │   │   └── ...
-    │   │
-    │   ├── agent            <- Model/agent-related modules.
-    │   │   └── ...
-    │   │
-    │   ├── core             <- Agent core modules.
-    │   │   └── ...
+    ├── discussion_agents                   <- Source code for this project.
+    │   ├── cog   
+    │   │   ├── agent                       <- Model/agent-related modules.
+    │   │   │   ├── base.py                 <-
+    │   │   │   ├── generative_agents.py
```

Comment:
don't need to specify every single file within a module. not scalable

Reply:
then i will just have the folders